src/promptflow-azure/promptflow/azure/_cli/_user_agent.py

```diff
@@ -2,4 +2,6 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 
-VERSION = "0.0.1"
+from promptflow.azure._version import VERSION
+
+USER_AGENT = "{}/{}".format("promptflow-azure-cli", VERSION)
```

Comment:
Are we exposing versions in the useragent? We may add more details in the description.

Reply:
Okay, let me make it.